### ISSUE TITLE
Dependabot won't create new labels via `default_labels` in `config.yml`

### ISSUE THREAD

**Issue by JacobEvelyn** [dependabot/feedback#935]
We just changed our `config.yml` to add a new label to `default_labels` and noticed that Dependabot wasn't adding it. It appears that it only works if the label already exists on the repo, which was surprising to us as we don't really manually use labels anywhere and Dependabot had just been creating them in the past (unless `dependencies` exists in all repos by default?).

**Comment by jurre** [comment]
Hi @JacobEvelyn, did the PR come from dependabot or dependabot-preview?

For context: dependabot is the github native version that handles automated security updates, and it is a separate app / config from dependabot-preview. So I want to rule out that the PR that is missing the labels is not coming from a different dependabot.

**Comment by JacobEvelyn** [comment]
Sorry—I'm referring to `dependabot-preview`!

**Comment by JacobEvelyn** [comment]
Ah, I just realized that `dependabot-preview` actually commented that it couldn't find the label. Still, if it could just create missing labels that would be amazing. See for example: https://github.com/panorama-ed/rapid-response/pull/70#issuecomment-629482114

**Comment by feelepxyz** [comment]
@JacobEvelyn yeah it's not super clear atm, had to dig through our code to figure out what we actually do. 

Dependabot only tries to create the default `dependencies` and `security` labels on the repo, along with language labels if you have configured multiple languages. Any extra labels you specify in your config file don't get automatically created if they don't exist.

We could possibly introduce a setting to force create all labels set in the config file but I'm not sure this is always the right thing to do and might cause issues if you've typoed a label that you delete and then gets re-created until you spot its set from the dependabot config file.

**Comment by JacobEvelyn** [comment]
Yeah I suspected you might not be creating labels specifically because of the typo case. I'm not sure what the best solution is—it's possible our use is an edge case, but it was definitely quite a pain for me to manually create the same label in ~40 repos. 🤷‍♀️
